middleware/logger/utils.go

```diff
@@ -37,3 +41,48 @@ func statusColor(code int, colors fiber.Colors) string {
 		return colors.Red
 	}
 }
+
+type customLoggerWriter struct {
+	loggerInstance fiberlog.AllLogger
+	level          fiberlog.Level
+}
+
+func (cl *customLoggerWriter) Write(p []byte) (int, error) {
+	switch cl.level {
+	case fiberlog.LevelInfo:
+		cl.loggerInstance.Info(utils.UnsafeString(p))
+	case fiberlog.LevelTrace:
+		cl.loggerInstance.Trace(utils.UnsafeString(p))
+	case fiberlog.LevelWarn:
+		cl.loggerInstance.Warn(utils.UnsafeString(p))
+	case fiberlog.LevelDebug:
+		cl.loggerInstance.Debug(utils.UnsafeString(p))
+	case fiberlog.LevelError:
+		cl.loggerInstance.Error(utils.UnsafeString(p))
+	default:
+		return 0, nil
+	}
+
+	return len(p), nil
+}
+
+// LoggerToWriter is a helper function that returns an io.Writer that writes to a custom logger.
+// You can integrate 3rd party loggers such as zerolog, logrus, etc. to logger middleware using this function.
+//
+// Valid levels: fiberlog.LevelInfo, fiberlog.LevelTrace, fiberlog.LevelWarn, fiberlog.LevelDebug, fiberlog.LevelError
+func LoggerToWriter(customLogger fiberlog.AllLogger, level fiberlog.Level) io.Writer {
```

Comment:
Could we name the first param `logger` ?

Reply:
`logger` would be okay for me